### ISSUE TITLE
Add a custom video player UI 

### ISSUE THREAD

**Issue by heysagnik** [heysagnik/screenREC#73]
![image](https://user-images.githubusercontent.com/70798495/196893692-f05aad56-428d-4195-8d43-e99bd4d1466d.png)

**Comment by LoganTann** [comment]
I don't understand this issue ??

**Comment by heysagnik** [comment]
> I don't understand this issue ??
I want a new redesigned video player UI. Currently we have the default video player is present. 

After Recording, you will find a preview of recording. I want to add a custom video player that matches the current design

**Comment by taimooralqureshi** [comment]
do you want a media player like the UI provided above? what is the current design; what is in your mind?

**Comment by heysagnik** [comment]
Yeah the provided UI would be better, other UI will also be good but don't use any third party modules

**Comment by LoganTann** [comment]
Oh I see. I think I will work on this issue

Then can you assign me ?

**Comment by heysagnik** [comment]
> Oh I see. I think I will work on this issue
> 
> Then can you assign me ?

Assigned you, would love to see what UI you are cooking 🐱
